framework/src/auxkernels/TagVectorAux.C

```diff
@@ -15,21 +15,28 @@ InputParameters
 TagVectorAux::validParams()
 {
   InputParameters params = TagAuxBase<AuxKernel>::validParams();
-  params.addRequiredParam<TagName>("vector_tag", "Tag Name this Aux works on");
-  params.addClassDescription("Couple a tag vector, and return its nodal value");
+  params.addClassDescription("Couple a tagged vector, and return its DOF value");
+  params.addRequiredParam<TagName>("vector_tag", "Name of the vector tag");
+  params.addParam<bool>("remove_variable_scaling", false, "Remove variable scaling from DOF value");
```

Comment:
```suggestion
  params.addClassDescription("Extract DOF values from a tagged vector into an AuxVariable");
  params.addRequiredParam<TagName>("vector_tag", "Name of the vector tag to extract values from");
  params.addParam<bool>("remove_variable_scaling", false, "Whether to remove variable scaling from DOF value. If false, values are directly extracted from the tag vector, and potentially with scaling applied. If true, any scaling of variables is undone in the reported values.");
```

Reply:
Maybe that's too wordy and might not completely explain it clearly still. I do think it's very important to explicitly spell out what is meant becuase it's easy to get mixed up about what scaling and unscaling are.